clients/fossid-webapp/src/main/kotlin/FossIdRestService.kt

```diff
@@ -209,7 +211,14 @@ interface FossIdRestService {
 
             val readTimeout = newReadTimeout?.toIntOrNull() ?: chain.readTimeoutMillis()
 
-            return chain.withReadTimeout(readTimeout, TimeUnit.MILLISECONDS).proceed(request)
+            return runCatching {
+                chain.withReadTimeout(readTimeout, TimeUnit.MILLISECONDS).proceed(request)
+            }.onFailure {
+                logger.error { "Error when intercepting the call chain: ${it.message}" }
+                val requestBuffer = Buffer()
+                request.body?.writeTo(requestBuffer)
```

Comment:
I assume, request bodies cannot contain sensitive information, right? So it should be safe to log them. Do all requests have a body?

Reply:
Argh you are right: FossID request bodies contain the api key :(
I removed this commit as I don't see an elegant workaround.
Maybe the `HttpLoggingInterceptor` of OkHttp: https://stackoverflow.com/a/56892146.